Mail/Components/RecipientField.swift

```diff
@@ -108,7 +106,14 @@ struct RecipientField: View {
         }
     }
 
+    private var trimmedInputText: String {
```

Comment:
Maybe we should move this variable above the functions?

Reply:
done